Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -152,9 +173,39 @@ extension SwiftLint {
             inputFiles["SCRIPT_INPUT_FILE_\(index)"] = file
         }
 
-        let outputJSON = shellExecutor.execute(swiftlintPath,
-                                               arguments: arguments,
-                                               environmentVariables: inputFiles)
+        return swiftlintViolations(swiftlintPath: swiftlintPath,
+                                   arguments: arguments,
+                                   environmentVariables: inputFiles,
+                                   outputFilePath: outputFilePath,
+                                   shellExecutor: shellExecutor,
+                                   failAction: failAction,
+                                   readFile: readFile)
+    }
+
+    static func swiftlintDefaultPath(packagePath: String = "Package.swift") -> String {
+        let swiftPackageDepPattern = "\\.package\\(.*SwiftLint.*"
+        if let packageContent = try? String(contentsOfFile: packagePath),
+            let regex = try? NSRegularExpression(pattern: swiftPackageDepPattern, options: .allowCommentsAndWhitespace),
+            regex.firstMatchingString(in: packageContent) != nil {
+            return "swift run swiftlint"
+        } else {
+            return "swiftlint"
+        }
+    }
+
+    private static func swiftlintViolations(swiftlintPath: String,
```

Comment:
Nitpick: swift-l-intViolations or swift-L-intViolations?

Reply:
The tool name is `swiftlint` :)
https://github.com/realm/SwiftLint/blob/master/Package.swift#L13